hendrix/test/performance/pyramid_with_crosstown_traffic.py

```diff
@@ -49,9 +49,10 @@ def cpu_heavy(heft, label=None):
             time.sleep(0)
 
         if count == end:
-            print("%s done: %s" % (label, time.time() - timer_start))
+            print "%s done: %s" % (label, time.time() - timer_start))
```

Comment:
I didn't mean to be picky, I was just browsing the code to see what may be in the works and happened to notice this.  I figured if I took 2 seconds to leave a comment it might save someone 5 minutes in the future.